File: src/main/java/com/spring/dto/response/HistoryListenResponse.java
package com.spring.dto.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class HistoryListenResponse {
    private String imageUrl;
    private String songName;
    private String message;
}

File: MSMA/app/src/main/java/vn/edu/usth/msma/data/dto/response/management/HistoryListenResponse.kt
package vn.edu.usth.msma.data.dto.response.management

import com.google.gson.annotations.SerializedName

data class HistoryListenResponse (
    @SerializedName("imageUrl")
    val imageUrl: String,

    @SerializedName("songName")
    val songName: String,

    @SerializedName("message")
    val message: String
)

